repository.go

```diff
@@ -8,122 +8,159 @@ import (
 	_ "github.com/lib/pq"
 )
 
-const (
-	host     = "localhost"
-	port     = 5432
-	user     = "postgres"
-	password = "1234"
-	dbname   = "test"
-)
-
-func getCart() {
-
-	psqlInfo := fmt.Sprintf("host=%s port=%d user=%s "+
-		"password=%s dbname=%s sslmode=disable",
-		host, port, user, password, dbname)
-	db, err := sql.Open("postgres", psqlInfo)
-	if err != nil {
-		panic(err)
-	}
-	defer db.Close()
+func getItem(db *sql.DB) {
+	sqlStatement :=
+		"select item_id, item_name, item_value " +
+			"from public.item " +
+			"where item_id = ?; "
+	sqlStatement = strings.Replace(sqlStatement, "?", "1", 1)
```

Comment:
You would also avoid this line if you use `fmt.Sprintf`, remember that you can use back quotes or back ticks for multiline string like

```
statement := fmt.Sprintf(`
    SELECT
        item_id,
        item_name,
        item_value,
    FROM public_item
    WHERE item_id = '%s';
`, param)
```